docs/Config.rst

```diff
@@ -72,6 +72,29 @@ GITHUB_ORG_NAME
 The name of your Github organization (the string in the URL whenever you
 go to the organization.
 
+GITHUB_DEFAULT_TEAM_NAME
+------------------------
+
+The name of the GitHub team in your organization that all users should
+be added to. Optional, defaults to ``all``.
+
+GITHUB_ADMIN_TEAM_NAME
+----------------------
+
+The name of the GitHub team in your organization that should be automatically
+promoted to Rocket administrators. Optional.
```

Comment:
```suggestion
promoted to Rocket administrators. Optional, defaults to ``exec``.
```

Reply:
convo in https://github.com/ubclaunchpad/rocket2/pull/504/files#r496301339